### ISSUE TITLE
clickhouse position function is transformed to `STR_POSITION` by `.sql` method

### ISSUE THREAD

**Issue by cpcloud** [tobymao/sqlglot#548]
```
In [8]: import sqlglot as sg

In [9]: sg.parse_one("position('a' in 'abc')", read="clickhouse").sql(dialect="clickhouse")
Out[9]: "STR_POSITION('abc', 'a')"
```

It should remain `position`, unfortunately clickhouse is case sensitive.

**Comment by cpcloud** [comment]
The argument order is incorrect too, but that's covered in #544.

**Comment by tobymao** [comment]
fixed by #550
